constance/codecs.py

```diff
@@ -54,6 +60,8 @@ def object_hook(o: dict) -> Any:
         if not codec:
             raise ValueError(f'Unsupported type: {o["__type__"]}')
         return codec[1](o['__value__'])
+    if isinstance(o, list) or (isinstance(o, dict) and not (set(o.keys()) & {'__type__', '__value__'})):
+        return o
```

Comment:
That's redundant
```
    if isinstance(o, list) or (isinstance(o, dict)
```
because we're in `object_hook` where o is `dict` by design

Reply:
https://github.com/Mogost/django-constance/commit/0566a7bc8361160be58fb16b62719cfd1d04222b